vulkancapsviewer.cpp

```diff
@@ -47,6 +47,10 @@
 #include <assert.h>
 #include <settingsDialog.h>
 #include "submitDialog.h"
+#if !defined(_WIN32) && !defined(__ANDROID__)

```

Comment:
Sorry for the late reply, but it looks like this may affect MacOS / iOS due to checking if it's not WIN32 and not ANDROID. Is there a a different way of checking if ustname needs to be used instead of Qt's built in functions?

Reply:
HI @SaschaWillems, I guess QT has some bug on BSD Unix platform, so we can use POSIX function to retrive OS' s informatiom or BSD specific function(bad way)

I suggest we should add an additional Macros:
`
#if !defined(_WIN32) && !defined(__ANDROID__) && !defined(__APPLE__)